wdl/TasksMakeCohortVcf.wdl

```diff
@@ -1004,5 +1004,83 @@ task ScatterVcf {
   >>>
   output {
     Array[File] shards = glob("~{prefix}.shard_*.vcf.gz")
+    Array[String] shards_string = glob("~{prefix}.shard_*.vcf.gz")
```

Comment:
Delete this since you won't need it in `SplitVcf.wdl` anymore. Also even though these are just strings, I'd be hesitant to put this into this widely used task since it would add a lot to the Cromwell footprint (metadata and database overhead).